test_cairo_contracts/test_valid_program.cairo

```diff
@@ -94,3 +94,12 @@ func test_array_sum_negative{output_ptr: felt*}() {
 
     return ();
 }
+
+// Infinite loop
+func test_infinite_loop_failing_test{
+    output_ptr: felt*
+}() {
+    %{ expect_revert() %}
+    test_infinite_loop_failing_test();
+    return();
+}
```

Comment:
Move this test closer to the feature itself.
```
/src/hooks/
    mod.rs
    tests.rs
    test_cairo_programs/
        infinite_loop.cairo
```

Example ici: https://github.com/open-dust/cairo-foundry/tree/main/src/hints/mock_call